### ISSUE TITLE
iOS 10 incompatibility?

### ISSUE THREAD

**Issue by mikehamil10** [VitaliiBlagodir/cordova-plugin-datepicker#215]
I'm unable to get the datepicker to appear on iOS 10. It works fine on Android and iOS 9 (and below), but NOTHING happens when trying to make the datepicker appear on iOS 10 devices.

Furthermore, no error message is thrown during compilation or at runtime :(
Anyone else experiencing this?

**Comment by paivaric** [comment]
Same problem

**Comment by mikehamil10** [comment]
I found a solution to this problem! My app is built on Ionic (v1) using the datepicker plugin via ngCordova.

I realized that NO cordova plugins were working and found that I had to update my Content-Security-Policy meta tag per instructions here: https://forum.ionicframework.com/t/cordova-geolocation-plugin-doesnt-work-with-ionic-on-ios-10/65363/3

Once doing that all plugins started working, including this one. So I feel bad for opening this issue (it's not specific to this plugin), it should be closed.
